hooks/nrpe_helpers.py

```diff
@@ -888,3 +881,69 @@ def set_netlinks_error():
     """Set the flag indicating a netlinks related error."""
     global NETLINKS_ERROR
     NETLINKS_ERROR = True
+
+
+def _get_cis_audit_check():
+    """Get the CIS audit step for nrpe.
+
+    When CIS audit is disabled no step is necessary.
+    """
+    if not hookenv.config("cis_audit_enabled"):
+        return {}
+
+    cis_cmd = cis_cmd_params(include_score=True)
+
+    cis_audit_check = {
+        "description": "Check CIS audit",
+        "cmd_name": "check_cis_audit",
+        "cmd_exec": local_plugin_dir + "check_cis_audit.py",
+        "cmd_params": cis_cmd,
+    }
+    return cis_audit_check
+
+
+def cis_misconfiguration():
+    """Check if CIS config are misconfigured.
+
+    if profile and tailoring file are used at the same time.
+    """
+    if hookenv.config("cis_audit_profile") and hookenv.config(
+        "cis_audit_tailoringfile"
+    ):
+        return True
+
+    return False
+
+
+def cis_tailoring_file_handler():
+    """Handle the tailoring file depending on the configuration.
+
+    When tailoring file is passed, creates a file at the default location with
+    the content passed on cis_audit_tailoringfile charm config.
+
+    Case tailoring file is not used, ensure that the file is erased
+    to not overlap with the profile passed.
+    """
+    if hookenv.config("cis_audit_tailoringfile"):
+        TAILORING_CIS_FILE.parent.mkdir(parents=True, exist_ok=True)
+        TAILORING_CIS_FILE.write_text(hookenv.config("cis_audit_tailoringfile"))
+        return
+
+    TAILORING_CIS_FILE.unlink(missing_ok=True)
+
+
+def cis_cmd_params(include_score=False):
+    """Get the CIS command parameters to use in the CLIs."""
+    cmd_params = []
+
+    profile = hookenv.config("cis_audit_profile")
+    cmd_params.append(f"-p {profile}" if profile else "")
+
+    cis_tailoring_file_handler()
+    tailoring = hookenv.config("cis_audit_tailoringfile")
+    cmd_params.append("-t" if tailoring else "")
+
+    if include_score:
+        cmd_params.append(hookenv.config("cis_audit_score"))
+
+    return " ".join(cmd for cmd in cmd_params if cmd)
```

Comment:
If the if expressions are no longer inlined above, we can also simplify this:

```suggestion
    return " ".join(cmd_params)
```